packages/native/src/lib/helpers/helpers.ts

```diff
@@ -0,0 +1,33 @@
+import { ReactTestInstance } from "react-test-renderer";
+
+/**
+ * Checks if a value is empty.
+ *
+ * @param value - The value to check.
+ * @returns `true` if the value is empty, `false` otherwise.
+ */
+export function isEmpty(value: unknown): boolean {
+  if (!value) {
+    return true;
+  }
+
+  if (Array.isArray(value)) {
+    return value.length === 0;
+  }
+
+  return false;
+}
+
+/**
+ * Converts a ReactTestInstance to a string representation.
+ *
+ * @param instance - The ReactTestInstance to convert.
```

Comment:
Nit: You don't need to add the hyphen. It's usually added automatically by other tools like VSCode 🙂 
```suggestion
 * @param instance The ReactTestInstance to convert.
```